<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataMenu</class>
 <widget class="QWidget" name="DataMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>593</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 211, 211);
QWidget{border-top-left-radius:15px;border-top-right-radius:5px;}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
stop:0#D3D3D3, stop: 1 #D3D3D3);
border:2px solid gray;
border-radius:5px;
margin-top:1ex;/* leave space at the top for the title */
}

QGroupBox::title {
subcontrol-origin: margin;
subcontrol-position: top left;
padding: 0 5px;
}</string>
   </property>
   <property name="title">
    <string>COM Setting</string>
   </property>
   <widget class="QPushButton" name="pushButtonConnectMcu">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 12pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rateCbx_plc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="rateCbx_stm">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>15</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="portCbx_plc">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="portCbx_stm">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>MCU</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>PLC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUsartStartFlush">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 12pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>Flush</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnectPlc">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 12pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1081</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_chartview"/>
  </widget>
  <widget class="QPushButton" name="pushButtonWaveReset">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>170</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
   </property>
   <property name="text">
    <string>Wave Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setBtnOutPutData">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>80</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setBtn">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>40</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTEST">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>530</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;黑体&quot;;
	color: rgb(0, 170, 0);
background-color:red;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 12pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{
font: 12pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>0</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditShow">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>461</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 9pt &quot;Arial&quot;;
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonTabel1">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>t1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonTabel2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>t2</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>10</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>TYPE</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_ABMagneticField">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>0</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_ZeroMagneticField">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>35</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>35</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPolarView">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>120</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;黑体&quot;;


	color: rgb(0, 170, 0);
background-color:gray;
color: white;   border-radius: 10px;  
border: 2px groove gray;                                            
border-style: outset;
}
QPushButton:hover{
font: 10pt &quot;黑体&quot;;
background-color:white; color: black;
}
QPushButton:pressed{font: 10pt &quot;黑体&quot;;
background-color:rgb(85, 170, 255);                                                           border-style: inset; 
}</string>
   </property>
   <property name="text">
    <string>PolarView</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxChart">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>220</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lock X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lock Y</string>
    </property>
   </item>
  </widget>
  <zorder>pushButtonTEST</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>pushButtonWaveReset</zorder>
  <zorder>setBtnOutPutData</zorder>
  <zorder>setBtn</zorder>
  <zorder>label_3</zorder>
  <zorder>textEditShow</zorder>
  <zorder>radioButtonTabel1</zorder>
  <zorder>radioButtonTabel2</zorder>
  <zorder>lcdNumber_ABMagneticField</zorder>
  <zorder>lcdNumber_ZeroMagneticField</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButtonPolarView</zorder>
  <zorder>comboBoxChart</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
